src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        int randomValue = new Random().nextInt(100);
```

Comment:
```suggestion
        int randomValue = random.nextInt(100);
```
move Random to field in this class (just like in ColoSupplier)